src/HotChocolate/ApolloFederation/src/ApolloFederation/Extensions/ProvidesDirectiveTypeDescriptorExtensions.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using HotChocolate.Language;
+using HotChocolate.Types;
+
+namespace HotChocolate.ApolloFederation.Extensions
+{
+    public static class ProvidesDirectiveTypeDescriptorExtensions
```

Comment:
This should be moved into ApolloFederationFieldDescriptorExtensions.

Reply:
Moved it to ApolloFederationFieldDescriptorExtensions